### PR TITLE
Spanish Translation: magic link emails for login

### PR DESCRIPTION
**Pull request by jhflorey**
This PR addresses issue https://github.com/episphere/connect/issues/1057

### CODE REVIEW

**Review by Gbarra9 — APPROVED** [review]
Looks good

File: utils/shared.js [hunk 1]
```diff
@@ -1540,10 +1540,27 @@ const filterSelectedFields = (dataObjArray, selectedFieldsArray) => {
...
 }
 
-const getTemplateForEmailLink = (email, continueUrl) => {
-    return `
+const getTemplateForEmailLink = (
+    email,
+    continueUrl,
+    preferredLanguage = fieldMapping.english.toString()
+) => {
+    return preferredLanguage !== fieldMapping.english.toString()
```
Comment (we-ai) [inline utils/shared.js:1548]: Bug can happen here: all values not equal to "English" will return Spanish email template, which is not ideal.

Comment (jhflorey) [reply utils/shared.js]: Solved

File: utils/notifications.js [hunk 2]
```diff
@@ -6,6 +6,7 @@ const {getResponseJSON, setHeadersDomainRestricted, setHeaders, logIPAdddress, r
 const {getNotificationSpecById, getNotificationSpecByCategoryAndAttempt, getNotificationSpecsByScheduleOncePerDay, saveNotificationBatch, updateSurveyEligibility, generateSignInWithEmailLink, storeNotification, checkIsNotificationSent, getNotificationSpecsBySchedule} = require("./firestore");
 const {getParticipantsForNotificationsBQ} = require("./bigquery");
 const conceptIds = require("./fieldToConceptIdMapping");
+const fieldToConceptIdMapping = require("./fieldToConceptIdMapping");
```
Comment (we-ai) [inline utils/notifications.js:9]:
```suggestion

```

Comment (jhflorey) [reply utils/notifications.js]: Solved

**Review by we-ai — APPROVED** [review]

**Pull request merged**